source/examples/exporting-csv.rst

```diff
@@ -0,0 +1,72 @@
+Export Tabular Data as a CSV
+****************************
+
+Problem
+=======
+
+Export data to a CSV file.
+
+Approach
+========
+
+Load the data from the databroker into a table (a DataFrame) and use the
+pandas package to effeciently export to CSV. Incorporate metadata in the
+filename.
+
+Example Solution
+================
+
+.. ipython:: python
+    :suppress:
+
+    from bluesky import RunEngine
+    from bluesky.plans import scan
+    from bluesky.examples import motor, det
+    import metadatastore.commands as mds
+    # import filestore.api as fs
+    # from databroker import Broker
+    # db = Broker(mds, fs)
+    from databroker import db
+    RE = RunEngine({})
+    RE.subscribe_lossless('all', mds.insert)
+
+We'll preface this example by running a scan to generate some example data.
+We'll generate three runs, destined for three separate CSV files.
+
+.. ipython:: python
+
+    from bluesky.plans import pchain  # a utility for chaining plans together
+
+    # This plan combines three 'scan' plans.
+    # Each scan includes some optional extra metadata that we'll use later.
+    master_plan = pchain(scan([det], motor, -10, 10, 15, md={'round': 1}),
+                         scan([det], motor, -5, 5, 15, md={'round': 2}),
+                         scan([det], motor, -3, 3, 10, md={'round': 3}))
```

Comment:
consider making an explicit point about the `round` metadata here. i did not pick up on this on my first read through and was confused where `round` came from in the csv output example. i thought it might have been a magic kwarg that `pchain` added. obviously that is not where it comes from... my mistake!